README.md

```diff
@@ -176,32 +188,43 @@ erDiagram
         string name2
         boolean enabled
     }
+    AreasProductionTypes {
+        int id PK
+        int area_id FK
+        int production_type_id FK
+    }
     Units {
         int id PK
         int area_id FK
         int production_type_id FK
+        int areas_production_type_id FK
         string internal_id
         string code
     }
     GenerationUnits {
         int unit_id PK,FK
-        timestamp time PK
+        timestamptz time PK
+        int value
+    }
+    GenerationData {
+        int areas_production_type_id PK,FK
+        timestamptz time PK
         int value
     }
     Generation {
```

Comment:
GenerationData replaces Generation? Don't think it makes sense to show both. But they are both queryable (views are read-only).